plugins/org.eclipse.elk.alg.layered/src/org/eclipse/elk/alg/layered/intermediate/PortSideProcessor.java

```diff
@@ -105,7 +105,7 @@ public static void setPortSide(final LPort port) {
         }
         
         // If the port anchor is not fixed, adapt it to its port side
-        if (!port.isAnchorFixed()) {
+        if (!port.isExplicitlySuppliedPortAnchor()) {
             KVector portSize = port.getSize();
             KVector portAnchor = port.getAnchor();
             
```

Comment:
Wouldn't we want this code to call something on `LPort` instead of computing sides ourselves? In fact, is this even still necessary?

Reply:
You are right this is not necessary, the anchor will be set by setSide in LPort.